src/main/java/com/epam/emotionalhelp/service/impl/QuizResultServiceImpl.java

```diff
@@ -13,10 +18,27 @@
 @Service
 @RequiredArgsConstructor
 public class QuizResultServiceImpl implements QuizResultService {
+    private final QuizResultRepository quizResultRepository;
+    private static final int PERCENT_COEFFICIENT = 20;
```

Comment:
```suggestion
    private static final int PERCENT_COEFFICIENT = 20;
    
    private final QuizResultRepository quizResultRepository;
```
Generally, we can declare constants - first and final fields - next.